apps/Deskreen/install-32

```diff
@@ -1,15 +1,16 @@
 #!/bin/bash
-mkdir .Deskreen || error 'Failed to make dir ".Deskreen"!'
-cd .Deskreen || error 'Failed to cd into .Deskreen!'
-#Download deskreen-1.0.1-armhf.deb from Discord
+
+# Download
+mkdir $HOME/.Deskreen || error 'Failed to create "$HOME/.Deskreen"!'
+cd $HOME/.Deskreen || error 'Failed to cd into .Deskreen!'
 wget https://raw.githubusercontent.com/Pi-Apps-Coders/files/main/Deskreen-1.0.1-armv7l.AppImage.zip || error 'Failed to download "Deskreen-1.0.1-armv7l.AppImage.zip" from Mirror!'
 unzip -o Deskreen-1.0.1-armv7l.AppImage.zip || error 'Failed to unzip Deskreen-1.0.1-armv7l.AppImage.zip!'
+
 # Install
-chmod +x Deskreen-1.0.1-armv7l.AppImage || error 'Failed to make Deskreen-1.0.1-armv7l.AppImage executable!'
+chmod +x $HOME/.Deskreen/Deskreen-1.0.1-armv7l.AppImage.zip || error 'Failed to make Deskreen-1.0.1-armv7l.AppImage executable!'
```

Comment:
This won't work, because deskreen is now unzipped, and we aren't running deskreen from the .zip.
```suggestion
chmod +x $HOME/.Deskreen/Deskreen-1.0.1-armv7l.AppImage || error 'Failed to make Deskreen-1.0.1-armv7l.AppImage executable!'
```